ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/wholeBodyHardwareControl/AvatarMultiThreadingFactory.java

```diff
@@ -176,10 +174,12 @@ public void stop()
 
    public void destroy()
    {
+      this.stop();
+      System.out.println("Calling destroy in the multi-threading factory");
```

Comment:
why would you change this from a LogTools call? You lose the stack trace using println, which makes it next to impossible to find where it's printing from.